ui/index.html

```diff
@@ -4,25 +4,25 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
-  <% if (typeof(VUE_APP_TITLE) !== 'undefined') { %>
-  <title><%= VUE_APP_TITLE %></title>
-  <% } else { %>
+<!--  <% if (typeof(VITE_APP_TITLE) !== 'undefined') { %>-->
+<!--  <title><%= VITE_APP_TITLE %></title>-->
+<!--  <% } else { %>-->
   <title>Monitoror</title>
-  <% } %>
-  <link rel="icon" type="image/png" href="<%= BASE_URL %>favicon.png">
-  <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+<!--  <% } %>-->
+  <link rel="icon" type="image/png" href="./favicon.png">
```

Comment:
You remove baseUrl? It's working with reverse proxy?

Like https://my.awesome.domaine.com/monitoror/